tests/core20/kernel/task.yaml

```diff
@@ -0,0 +1,5 @@
+summary: Ensure that we have the right kernel
+
+execute: |
+    echo "Check kernel version"
+    uname -r | MATCH "^5.4"
```

Comment:
what's the need for this test? does core20 have a hard requirement on the 5.4 kernel ?

Reply:
good question. @mvo5 is this test needed for core20 as well?